x-pack/plugins/security_solution/server/features.ts

```diff
@@ -116,16 +116,17 @@ export const getKibanaPrivilegesFeaturePrivileges = (ruleTypes: string[]): Kiban
   subFeatures: [],
   privileges: {
     all: {
-      app: [APP_ID, 'kibana'],
+      app: [APP_ID, 'csp', 'kibana'],
```

Comment:
I suggest creating a constant in this file for the string `csp` and mention in a comment that it needs to match the cloud security posture plugin ID